checkov/common/output/common.py

```diff
@@ -15,3 +15,32 @@ class ImageDetails(SCADetails):
     image_id: str = ''
     name: str | None = ''
     related_resource_id: str | None = ''
+
+
+def is_raw_formatted(licenses: str) -> bool:
+    return licenses.__contains__('","')
+
+
+def format_licenses_to_string(licenses_lst: list[str]) -> str:
+    print('format_licenses_to_string - start', licenses_lst)
+    if licenses_lst and len(licenses_lst) > 1:
+        joined_str = '","'.join(licenses_lst)
+        return f'"{joined_str}"'
+    elif licenses_lst and len(licenses_lst) == 1:
+        return licenses_lst[0]
+    else:
+        return 'Unknown'
+
+
+def format_string_to_licenses(licenses_str: str) -> list[str]:
+    print('format_string_to_licenses - start', licenses_str)
+    if licenses_str == 'Unknown':
+        return [licenses_str]
+    elif licenses_str and len(licenses_str) > 0:
```

Comment:
```suggestion
    elif licenses_str:
```
no need for the right side, because the left side checks if it is not empty